node/cmd/guardiand/node.go

```diff
@@ -571,8 +581,13 @@ func runNode(cmd *cobra.Command, args []string) {
 		if *bigTableTopicName == "" {
 			logger.Fatal("Please specify --bigTableTopicName")
 		}
-		if *bigTableKeyPath == "" {
-			logger.Fatal("Please specify --bigTableKeyPath")
+		if *guardianCredentialsFile == "" {
```

Comment:
Q: Is `guardianCredentialsFile` a renaming of `bigTableKeyPath`?
* if no, then why do we want to remove `bigTableKeyPath`?

Reply:
Great question. Initially, my thinking here is that instead of managing & provisioning multiple keys in the Guardian node, we could use one key and give it
1. Read/write permission to big table
2. KMS signing permission

That was when I renamed it to `credentialsFile` since it becomes more general now. Then I realized that it is possible that we don't need to provision keys at all at the guardian node level (e.g. [here](https://github.com/alephium/wormhole-fork/blob/d0706cf5dee3fab61296d5c2f385686ff83a2e2f/node/pkg/ecdsasigner/cloudkms.go#L30) we didn't specify key file for KMS, we can discuss the detail offline). I will see if I can remove the need to specify key file for big table. If that is the case then we can remove this flag all together and it would simplify infra and arguably more secure. 
WDYT?

Reply:
`guardianCredentialsFile` is removed now for the keyless solution we discussed yesterday, tested that bigtable works.